core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -2511,7 +2511,7 @@ public void unmountNonExistingPathWithException() throws Exception {
    */
   private AlluxioURI createTempUfsDir(String ufsPath) throws IOException {
     String path = mTestFolder.newFolder(ufsPath.split("/")).getPath();
-    return new AlluxioURI(path);
+    return new AlluxioURI("file:??", null, path);
```

Comment:
??

Reply:
this should be just `file` for the scheme?

Reply:
🤦‍♂️ 🤦‍♂️  sticky shift keys....